src/FixedDataTable.js

```diff
@@ -1204,7 +1204,9 @@ var FixedDataTable = createReactClass({
     );
 
     var lastScrollToColumn = oldState ? oldState.scrollToColumn : undefined;
-    if (props.scrollToColumn !== null && props.scrollToColumn !== lastScrollToColumn) {
+    if (props.scrollToColumn !== null 
+        && props.scrollToColumn !== lastScrollToColumn
+        && columnInfo.bodyScrollableColumns.length>0) {
```

Comment:
Can you add spaces around the > operator to fit our styles?

After that, I'll merge and cut a point release.  Thanks